package.json

```diff
@@ -4,7 +4,10 @@
   "description": "A git binary installed through NPM, for use with AWS Lambda",
   "main": "index.js",
   "scripts": {
-    "test": "bats test.sh"
+    "lint": "eslint .",
+    "prepublishOnly": "gulp; npm test",
```

Comment:
I think this should probably be `gulp && npm test`. With a `;` if the first command fails it doesn't affect the second or the resulting error code. E.g. `false; echo 'test'` prints 'test' and exits with 0, while `false && echo 'hi'` prints nothing and exits with 1.

Reply:
done - replaced ; with &&